packages/database/prisma/schema.prisma

```diff
@@ -202,6 +202,8 @@ model Purchase {
   redeemedBulkCouponId String?
   productId            String
   merchantChargeId     String?
+  quantity             Int?
+  checkoutSessionId    String?
```

Comment:
this should be `identifier` in new `MerchantPurchase` table to fit with existing modeling

I think quantity might belong there as well, unless we can make a strong case for leaving it at this layer.